topi/python/topi/hls/nn.py

```diff
@@ -0,0 +1,407 @@
+# pylint: disable=invalid-name,unused-variable,unused-argument
```

Comment:
i think it's more reasonable to put these in the functions, where you intentionally wanted to disable the linter.

Reply:
good call on the linter directives

Reply:
@liangfu @tmoreau89 This PR has been already merged, so I think of addressing the comment in future cleanup patch.  Is it fine with you?

Reply:
sure